en/django_urls/README.md

```diff
@@ -93,10 +93,23 @@ The last part, `name='post_list'`, is the name of the URL that will be used to i
 
 If you try to visit http://127.0.0.1:8000/ now, then you'll find some sort of 'web page not available' message. This is because the server (remember typing `runserver`?) is no longer running. Take a look at your server console window to find out why.
 
-![Error](images/error1.png)
+{% filename %}{{ warning_icon }} command-line{% endfilename %}
+
+```shell
```

Comment:
I'm not sure this really is shell.

Reply:
It's actually stdout, as it's generated by the `runserver` command, but I don't think we will get any syntax highlighting, no matter the language we specify here. Are we OK with it just being ```?

Reply:
Yes, I think that's fine.